scss/_patterns_suru.scss

```diff
@@ -129,4 +130,32 @@
       background-position: top right;
     }
   }
+
+  .p-suru--divider {
+    background-color: var(--vf-suru-background);
+    color: $colors--theme--text-default;
+    // same as %section-padding--deep
+    padding-bottom: calc($spv--strip-deep / 2);
+    position: relative;
+
+    // only apply suru background images on large screens
+    @media (min-width: $breakpoint-large) {
+      padding-bottom: calc(2 * $spv--strip-deep);
+
+      &::after {
+        background-image: var(--vf-suru-divider);
+        background-position: bottom left;
+        background-repeat: no-repeat;
+        background-size: 1280px 237px; // based on image dimensions (2560x474) scaled down by 0.5
```

Comment:
Don't we usually use `rems` instead of `px`?

Reply:
Yes, but these dimensions are directly related to size of the image (in pixels) - we don't want to scale it in any way. I think… actually, a good question. I'll confirm with Lyubo.